crates/sui-indexer/migrations/2022-11-28-204251_addresses/up.sql

```diff
@@ -1,7 +1,12 @@
+-- SuiAddress and ObjectId type, 0x + 64 chars hex string
+CREATE DOMAIN address VARCHAR(66);
+-- Max char length for base58 encoded digest
+CREATE DOMAIN base58digest VARCHAR(44);
```

Comment:
I am not sure if it is a good idea to hardcode the lengths. Are we certain that these will never change again?
cc @joyqvq @kchalkias @lxfind

Reply:
I don't think this can change again :| , however event if the key length change we can alter the domain really easily in DB